lib/appsignal/system.ex

```diff
@@ -2,4 +2,11 @@ defmodule Appsignal.System do
   def heroku? do
     System.get_env("DYNO") != nil
   end
+
+  def root? do
+    case System.cmd("id", ["-u"]) do
```

Comment:
For this scenario, probably, assuming the `root` user is always named `root`.

But the `uid` is also used for another part of the diagnose command. In the "Paths" section we compare the owner of paths and files in the "paths" section to debug the ownership of the "appsignal paths". That's why we need to know the `uid` number. `Appsignal.System.root?/0` can remain, but we also need a more direct function to receive the `uid`, such as `Appsignal.System.uid/0`

Side-note: Not finding a good way to uncover the `uid` for the process I added the `Process user` entry to the diagnose output which does just that. Now that we've decided on a way to fetch the `uid` from the system by shelling out, the "Process user" output line can be removed.